app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.update(comments_counter: post.posts_counter + 1)
```

Comment:
- [ ] Here Instead of `post.posts_counter` it should be `post.comments_counter`.